symphony/bdk/core/auth/auth_session.py

```diff
@@ -0,0 +1,34 @@
+class AuthSessionRSA:
+    """
+    RSA Authentication session handle.
+    """
+
+    def __init__(self, authenticator):
+        self.__session_token = None
```

Comment:
Please use a single underscore. For the meaning of single and double underscores in python: https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name